tekton-example/README.md

```diff
@@ -26,52 +30,84 @@ Creation of a TaskRun (using `promote-run.yaml`) will then further promote from
 ## Create Tekton Resources
 
 - Create a new namespace:
+
 ```shell
 kubectl create ns <namespace>
 ```
 
 - Modify your Kubernetes context to use this namespace by default:
-```shell 
+
+```shell
 kubectl config set-context --current --namespace=<namespace>
 ```
 
 - Apply the resources folder:
-```shell 
+
+```shell
 kubectl apply -f resources
```

Comment:
It's a nit - but you didn't tell me I need to choose v1alpha1/v1beta1, then use kubectl apply -f v1beta1 (or v1alpha1)/resources. We could tell users to just cd into the v1alpha1 or v1beta1 directory, otherwise we need to qualify all subsequent commands